src/program/lwaftr/run/run.lua

```diff
@@ -68,24 +68,26 @@ function parse_args(args)
          fatal('Failed to enable real-time scheduling.  Try running as root.')
       end
    end
-   function handlers.n(arg)
-      v4_pci = arg
+   function handlers.v4(arg)
+      v4 = arg
       if not arg then
-         fatal("Argument '--v4-pci' was not set")
+         fatal("Argument '--v4' was not set")
       end
-      if not nic_exists(v4_pci) then
-         fatal(("Couldn't locate NIC with PCI address '%s'"):format(v4_pci))
+      if not nic_exists(v4) then
+         fatal(("Couldn't locate NIC with PCI address '%s'"):format(v4))
       end
    end
-   function handlers.m(arg)
-      v6_pci = arg
-      if not v6_pci then
-         fatal("Argument '--v6-pci' was not set")
+   handlers["v4-pci"] = handlers.v4
+   function handlers.v6(arg)
+      v6 = arg
+      if not v6 then
+         fatal("Argument '--v6' was not set")
       end
-      if not nic_exists(v6_pci) then
-         fatal(("Couldn't locate NIC with PCI address '%s'"):format(v6_pci))
+      if not nic_exists(v6) then
+         fatal(("Couldn't locate NIC with PCI address '%s'"):format(v6))
       end
    end
+   handlers["v6-pci"] = handlers.v6
```

Comment:
LGTM.  Would you mind making --v4-pci and --v6-pci print a warning before dispatching to the v4/v6 handler?  That will get our users migrating to the new argument name.  Thank you!